testsuite/src/it/java/io/quarkus/ts/startstop/utils/WhitelistLogLines.java

```diff
@@ -79,7 +79,7 @@ public enum WhitelistLogLines {
             // https://github.com/quarkusio/quarkus/pull/28810
             Pattern.compile(".*Stream is closed, ignoring and trying to continue.*"),
             // https://github.com/quarkusio/quarkus/pull/28654
-            Pattern.compile(".*Using legacy gRPC support, with separate new HTTP server instance. Switch to single HTTP server instance usage with quarkus.grpc.server.use-separate-server=false property.*"),
+            Pattern.compile(".*Using legacy gRPC support, with separate new HTTP server instance. This is the current default to maintain compatibility.*"),
```

Comment:
I think it should be changed same as https://github.com/quarkus-qe/quarkus-startstop/pull/518/files , but it it's passing I dno't have problem